demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,209 @@
+                continue
+
+            # extract content items type, for example: Integrations, Scripts, Playbooks, etc.
+            content_type = section[0]
+
+            # extract content items, for example: HelloWorld, HelloWorldScript, HelloWorldPlaybook, etc.
+            content_items = section[1]
+            content_items_sections_ls = CONTENT_ITEM_SECTION.findall(content_items)
+
+            for content_type_section in content_items_sections_ls:
+                content_type_section = remove_none_values(ls=content_type_section)
+                if content_type_section:
+                    logger.debug(
+                        f'removing New: " if "New:" in {content_type_section[0]} else "" '
+                    )
+                    header = (
+                        content_type_section[0].rstrip()
+                        if ("New: " not in content_type_section[0])
+                        else content_type_section[0][len("New: ") :]
+                    )
+                    headers.setdefault(content_type, []).append(header)
+        return headers
+
+    def validate_content_type_header(self, header: str) -> bool:
+        """
+            Validate that the release notes 1st headers (the content type) are a valid content entity.
+        Args:
+            content_type: (str) - The content type to validate.(e.g. Integrations, Playbooks, etc.)
+        Return:
+            True if the content type is valid, False otherwise.
+        """
+        return header in RN_HEADER_BY_FILE_TYPE.values()
+
+    def validate_content_item_header(
+        self,
+        headers_to_display_names: Dict[str, List[str]],
+        pack_items_by_types: Dict[ContentType, List[ContentItem]],
+    ) -> Dict[str, Set[str]]:
+        """
+        Validate that the content items' display names match expected values.
+
+        Args:
+        headers_to_display_names: A dictionary mapping header names (e.g., Integrations)
+            to lists of expected display names for content items.
+        pack_items_by_types: A dictionary mapping content types to lists of ContentItem
+            instances.
+
+        Returns:
+            A dictionary where keys are ContentType and values are sets of invalid display names
+            for content items that do not match expected values.
+        """
+        missing_display_names = {}
+        for header, display_names in headers_to_display_names.items():
+            logger.debug(f"Validating {header} content items")
+            content_type = ContentType.from_release_note_header(header)
+            pack_display_names = {
+                item.display_name for item in pack_items_by_types.get(content_type, [])
+            }
+            if missing := set(display_names).difference(pack_display_names):
+                missing_display_names[header] = missing
+        return missing_display_names
+
+    def validate_release_notes_headers(
+        self, content_item: Pack
```

Comment:
```suggestion
        self, pack: Pack
```
easier to read since we know it's a pack